stacks/ibm/components/seldon-core-operator/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../seldon/seldon-core-operator/overlays/application
```

Comment:
Since a `stacks/ibm/application/seldon-core-operator` exists, should that be referenced here instead?

Reply:
@pvaneck I was debating whether `seldon` should go with components as it was part of the `stacks/ibm/kustomization.yaml` when we first created the full stack version. Then the thought goes that `seldon` may not necessarily a component of Kubeflow, so I would like to provide it as an application. So I ended up providing both kustomization.yaml file. At some point in time we may remove seldon from both the full stack and the components.